Session Replay Q2 Objectives (#8103)
PostHog/posthog.com: contents/teams/replay/objectives.mdx

### 🎯 Goal 1: Quality and Consolidation

Why? 

We've done a lot of investigation and preparation work and we want to captilise on that

* Mobile replay out of beta for Android/iOS
* Build the best UX for capturing and playing recordings
* Build the best UX for finding recordings

### 🎯 Goal 2: Error tracking MVP

> Why? PostHog is "The single platform to analyze, test, observe, and deploy new features". Having network timings, network payloads, and console errors in Session Replay is a great developer tool. We want to learn by doing to understand how we could capture and expose even more information here to help people dig in to what is happening with their products

Reviewer: I'd define a few tasks here since even error tracking as MVP is a huge milestone already.

A few things that come to my mind.

* SDK work (capture API, error call back hook up - this is per framework, source maps linking, SDK bundle size).
* Symbols uploading and linking (source maps for now) - need a CLI
* Ingestion (probably reusing `$exception`)?
* Symbolication
* Grouping - signal vs noise
* Frontend Issues list - read, resolve, etc
* Alerting

We can define what our MVP is gonna do and list them?

Reviewer: yeah probably not needed, you're right, just a braindump

> Follow along in [our Q2 MegaIssue](https://github.com/PostHog/posthog/issues/21078)

Reviewer: I'd move this up or down (before goal 1 or after goal 3) otherwise it feels only related to the goal 2

Author: Lazy editing 🙈

> ### 🎯 Goal 1: Quality and Consolidation

Reviewer 2: ```suggestion
### 🎯 Goal 1: Quality and Consolidation for Session Replay (web & mobile)
```

Why? 

We've done a lot of investigation and preparation work and we want to capitalize on that

* Mobile replay out of beta for Android/iOS

> * Build the best UX for capturing and playing recordings

Reviewer 3: I wonder if this fully captures what we want to achieve from a capture / playback point of view. Maybe something like:
```suggestion
* Make capture and playback as complete, performant and easy as possible
```

> * Build the best UX for finding recordings

Reviewer 3: ```suggestion
* Build the best UX for finding and playing recordings
```

### 🎯 Goal 2: Error tracking MVP

Why? PostHog is "The single platform to analyze, test, observe, and deploy new features". Having network timings, network payloads, and console errors in Session Replay is a great developer tool. We want to learn by doing to understand how we could capture and expose even more information here to help people dig into what is happening with their products

* Building on top of the existing `$exception` ingestion

> * In the hands of real users getting feedback on solving their problems

Reviewer 2: ```suggestion
* Getting it in the hands of real users, so that we can collect feedback on how to best solve their problems
```

-> the wording makes more sense to me this way
